### PR TITLE
MRG: add scaled FAQ, adjust ksize answer

### LINKED ISSUE #2918: provide/update/refactor documentation around k-mer sizes and scaled
per [@LilyAnderssonLee review](https://github.com/pyOpenSci/software-submission/issues/129#issuecomment-1838233643)

>I may have overlooked it in your documentation. It would be helpful if you could provide some recommendations for the selection of `k-mers` and `scales`.

and

>What are the consequences of using different scales? Do you recommend using the same scale to generate the signatures for genomes or databases?

### PR DESCRIPTION
**Pull request by ctb**
Adds the following FAQ entry to address https://github.com/sourmash-bio/sourmash/issues/2918:

> ## What scaled values should I use with sourmash?
> 
> We recommend scaled=1000 or scaled=10000 when working with bacterial
> and archaeal sketches and DNA. We have quite a bit of experience with
> this, and even some
> [published benchmarks](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-022-05103-0)
> showing that this works very well.  You may need to use lower scaled
> values with smaller query and target sequences, such as viral genomes
> or genes, but we do not have systematic advice on this.
> 
> That having been said, you can always use a lower scaled value - the only
> consequence is that memory and compute requirements increase.
> 
> Also, sourmash will automatically use the larger of two scaled values
> when comparing two sketches with different scaled values. So if, for example,
> you use [the precomputed databases](databases.md), you will always end up
> using your query sketches at a minimum scaled of 1000, even if you created
> them with a lower scaled value.
> 
> Please also see [What resolution should my signatures be?](using-sourmash-a-guide.md#what-resolution-should-my-signatures-be-how-should-i-create-them).

Fixes https://github.com/sourmash-bio/sourmash/issues/2918

### CODE REVIEW

**Comment by ctb** [comment]
Ready for review @sourmash-bio/devs

File: doc/faq.md [hunk 1]
```diff
@@ -113,12 +113,34 @@ and k=51 are negligible; and that (b) k=31 works fine for most
 day-to-day use of sourmash. 
 
 We also provide [Genbank and GTDB databases](databases.md) for k=21,
-k=31, and k=51.
+k=31, and k=51, so choosing those k-mer sizes for your own sketches
```
Comment (ccbaumler) [inline doc/faq.md]:
```suggestion
k=31, and k=51, so choosing from those k-mer sizes for your own sketches
```

**Review by ccbaumler — APPROVED** [review]
LGTM

**Comment by ccbaumler** [comment]
An additional FAQ about the relationship between `threshold-bp` and `scaled`could be useful as well

**Comment by ctb** [comment]
> An additional FAQ about the relationship between `threshold-bp` and `scaled`could be useful as well

create an issue?

And note that this only refers to prefetch and gather :)

**Pull request merged**